Popularity/DCAFPilot/src/python/DCAF/ml/model.py

```diff
@@ -198,24 +206,67 @@ def model(train_file, newdata_file, idcol, tcol, learner, lparams=None,
 
     # new data file for which we want to predict
     if  newdata_file:
-        tdf = read_data(newdata_file, drops, scaler=scaler)
-        if  tcol in tdf.columns:
-            tdf = tdf.drop(tcol, axis=1)
-        if  verbose:
-            print("New data file", newdata_file)
-            print("Columns:", ','.join(tdf.columns))
-            print("test shapes:", tdf.shape)
-        datasets = [int(i) for i in list(tdf['dataset'])]
-        #20160301 'dbs' header changed to 'dbsinst'
-        dbs_h = 'dbsinst' if 'dbsinst' in tdf.keys() else 'dbs'
-        dbses = [int(i) for i in list(tdf[dbs_h])]
-        if  scaler:
-            tdf = getattr(preprocessing, scaler)().fit_transform(tdf)
-        predictions = fit.predict(tdf)
-        data = {'dataset':datasets, dbs_h: dbses, 'prediction':predictions}
-        out = pd.DataFrame(data=data)
-        if  ofile:
-            out.to_csv(ofile, header=True, index=False)
+        nfiles = []
+        if  not ',' in newdata_file:
```

Comment:
Can you make it more generic via glob such that input files can be either separated by comma or provided as a pattern, e.g. file*.csv